src/azure-cli-core/azure/cli/core/application.py

```diff
@@ -100,6 +100,8 @@ def execute(self, unexpanded_argv):
             self.session['command'] = expanded_arg.command
             try:
                 _validate_arguments(expanded_arg)
+            except CLIError:
```

Comment:
But on the next line you raise the previous exception (CLIError) anyway? I guess I'm missing something. 😕